.buildkite/scripts/serverless/create_deploy_tag/create_fix_tag.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -euo pipefail
+
+git tag "$BUILDKITE_BRANCH"
```

Comment:
This will just tag `HEAD`. We need to ensure that we specifically tag the specified commit (I think that's `BUILDKITE_COMMIT`?). Preferably I want to also verify that this commit is part of the specified `deploy-fix@*` branch, so the two actually belong together.

Reply:
I pushed https://github.com/elastic/kibana/pull/176258/commits/f344641d7f3b2445bfcb64fe6496dfb853766ee7 + https://github.com/elastic/kibana/pull/176258/commits/39c26403f693a13070bdc8507c9b156db512970d